Examples/LambdaFunctions/README.md

```diff
@@ -18,7 +18,9 @@ Steps to deploy this sample to AWS Lambda using the AWS CLI:
   ./scripts/deploy.sh
   ```
 
-  Note: This script assumes you have AWS CLI installed and credentials setup in `~/.aws/credentials`.
+  Notes: 
+  - This script assumes you have AWS CLI installed and credentials setup in `~/.aws/credentials`.
+  - Update `s3_bucket=swift-lambda-test` in `deploy.sh` before running (AWS S3 buckets require a unique global name)
```

Comment:
this is helpful. we may want to point out both  the bucket and function names in the deploy script

Reply:
In my opinion the S3 bucket selection should be in the script, not just in the README.md.
The issue is that the S3 name must be globally unique so a predefined one works only for the owner of it.

Reply:
@Andrea-Scuderi, that is a valid point, but I think outside of the scope of this PR.

I think there are two topics to cover:

1. Help new users, unfamiliar with Lambda, get started quickly with these examples. This was the goal for this PR. Maybe documentation within the script would be helpful too.

2. Make deployment script(s) more _robust_, so it's easy to deploy one or more lambdas within a package with one command. This could be a separate PR, maybe even a separate repo (would be nice to have swift cli tool included to do this).

Focusing on the first point, I'll update the PR to indicate how to update both bucket and lambda function name before deploying.

Reply:
would be good to track any future ideas as illustrated ^^ in GitHub issues so we dont loose track of them